plans/PROGRESS.md

```
@@ -3,6 +3,17 @@
 This is an evidence log, not an authorization to merge or deploy. Update it at
 each completed plan gate.
 
+### Course correction W3a: matchup Markov estimator (ADR-271) — 2026-08-29
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)